PaymentRails/ruby-sdk
# [Testing] Add characterization testing to gateways missing coverage

# Fixes #

1. Add characterization testing to uncovered gateways
2. Update gateways to return object upon update, rather than boolean

### To test:
1. Set an `.env` file. See `.env.template` for an example
3. Run integration testing.  To avoid rate limiting, try one by one, e.g. `bundle exec rake integration_tests TEST='spec/integration/BalanceTest.rb'`

### Checklist

- [x] I have made a material change to the repo (functionality, testing, spelling, grammar)
- [x] I have titled the PR appropriately
- [x] I have updated my branch with the main branch
- [ ] I have added the necessary documentation about the functionality in the appropriate .md file
- [ ] I have added inline documentation to the code I modified

If you have questions, create a GitHub Issue in this repository.
